cotton/salt/pillar.py

```diff
@@ -83,30 +101,81 @@ def get_rendered_pillar_location(pillar_dir=None, projects_location=None, parse_
                     # a variable. This would make the filename start with / and instead of
                     # writing under dest_location it will try to write in /
                     if isinstance(file_short, str):
-                        files_to_render.append('./' + file_short.replace('.', '/') + '.sls')
+                        files_to_render.append(file_short.replace('.', '/') + '.sls')
     else:
         # let's select all files from pillar directory
-        for root, dirs, files in os.walk(pillar_dir):
-            rel_path = os.path.relpath(root, pillar_dir)
-            for file_name in files:
-                files_to_render.append(os.path.join(rel_path, file_name))
+        for pillar in pillars:
+            for root, dirs, files in os.walk(pillar):
+                rel_path = os.path.relpath(root, pillar)
+                for file_name in files:
+                    files_to_render.append(os.path.join(rel_path, file_name))
+
+    if __render_templates(files_to_render, dest_location, jinja_env) is False:
+        print(red("Aborting due to pillar failing to render"))
```

Comment:
@tim-mcarthur-moj  Nope, they where marked above as failures, have added it more explicitly now

Sample output

```
Pillar could not compile the following templates:
 - ./foo.sls
 - ./bar.sls
```